src/tools/generateOptionsSchema.ts

```diff
@@ -357,6 +357,9 @@ export function GenerateOptionsSchema() {
     packageJSON.contributes.debuggers[1].configurationAttributes.launch = schemaJSON.definitions.LaunchOptions;
     packageJSON.contributes.debuggers[1].configurationAttributes.attach = schemaJSON.definitions.AttachOptions;
 
+    // Mono
+    packageJSON.contributes.debuggers[2].configurationAttributes.launch = schemaJSON.definitions.LaunchOptions;
```

Comment:
I think instead we probably don't want to declare what options `monovsdbg` takes since we don't want people using it directly.